### ISSUE TITLE
[feature] cache RPMS

### ISSUE THREAD

**Issue by GeoffWilliams** [declarativesystems/pdqtest#25]
Downloading via YUM takes ages - suggestion from customer - cache /var/cache/yum on the host - so simple!

- thanks Angus!

**Comment by GeoffWilliams** [comment]
Feature implemented in master - initial times against forge module geoffwilliams-ssh which only install openssh.  If your downloading more you will see a larger benefit, if you download less packages you will see less benefit 

1st run no cache yet
```
real    0m58.484s
user    0m8.024s
sys     0m0.812s
```

2nd run (cached)
```
real    0m39.875s
user    0m8.360s
sys     0m0.716s
```

Or 32% less time needed :) -- in ef105ee825cc1853f8b55be2675ed9cb409f5359

**Comment by GeoffWilliams** [comment]
released in 0.6.0
